irods/manager/data_object_manager.py

```diff
@@ -534,6 +534,37 @@ def make_FileOpenRequest(**extra_opts):
             ret_value.seek(0,io.SEEK_END)
         return ret_value
 
+    def replica_truncate(self, path, desired_size, **options):
+            
+        if self.sess.server_version == (4,3,2):
+            message = 'replica_truncate responses can fail to parse with iRODS 4.3.2 due to routine omission of the JSON response string.  See issue: irods/irods#7918'
```

Comment:
Let's keep github issue numbers out of the error messages.

The additional wording sounds good to me.

I wonder if it would be better to state that the method isn't supported first and then give the reason. I don't have strong feelings on that. Keeping the suggestion or swapping the order accomplish the job.

Reply:
Yep. That's it.

I defer to you and @d-w-moore.